website/addons/wiki/utils.py

```diff
@@ -154,3 +155,64 @@ def format_wiki_version(version, num_versions, allow_preview):
         raise InvalidVersionError
 
     return version
+
+def serialize_wiki_settings(user, node_ids):
```

Comment:
The first time it is called it takes a single node, but when it is called recursively it receives a list.

        children.extend(serialize_wiki_settings(
            user,
            [
                n._id
                for n in node.nodes
                if n.primary and
                not n.is_deleted
            ]
        ))